playbooks/app_install/roles/docker_install/vars/main.yml

```diff
@@ -1,15 +1,35 @@
 ---
 # vars file for docker_install
-# Which docker version should be installed
-docker_version: '23.0.6'
+docker_version: 24.0.6
+# Whether to retrieve from artifactory to push to target
+# or push from the controller to the target when set local
+docker_rpm_src: artifactory
+artifact_path:  "https://engci-maven-master.cisco.com/artifactory/cmsp-sapphire-development-group/up-sac-utils/jenkins/v1/docker-ce-24.0.6-rpms.tar.gz"
 # Which packages should be cleared out before installing docker
 package_removals:
   - podman
   - containers-common
+  - containerd
+  - containerd.io
   - runc
-  - docker-ce-cli
   - docker
+  - docker-engine
   - docker-ce
-  - containerd.io
+  - docker-ce-cli
   - docker-buildx-plugin
   - docker-compose-plugin
+  - docker-client
+  - docker-client-latest
+  - docker-common
+  - docker-latest
+  - docker-latest-logrotate
+  - docker-logrotate
+
```

Comment:
Hmm, I would expect all these removals to negate the need to delete the individual files in docker_install/tasks/main.yml file, no?